object.go

```diff
@@ -456,6 +461,7 @@ func (o *Object) Download(opts *RequestOptions) DownloadedObject {
 	}.Do(o.c.a.backend)
 	var body io.ReadCloser
 	if err == nil {
+		defer resp.Body.Close()
```

Comment:
This is incorrect. This function moves `resp.Body` into the returned DownloadedObject.